tests/admintools.test.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+require('should');
+var _ = require('lodash');
+var benv = require('benv');
+var read = require('fs').readFileSync;
+var serverSettings = require('./fixtures/default-server-settings');
+
+var nowData = {
+  sgvs: [
+    { mgdl: 100, mills: Date.now(), direction: 'Flat', type: 'sgv' }
+  ]
+};
+
+var someData = {
+  '/api/v1/devicestatus.json?count=500': [
+    {
+    "_id": {
+        "$oid": "56096da3c5d0fef41b212362"
+    },
+    "uploaderBattery": 37,
+    "created_at": "2015-09-28T16:41:07.144Z"
+    },
+    {
+    "_id": {
+        "$oid": "56096da3c5d0fef41b212363"
+    },
+    "uploaderBattery": 38,
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1026365146/issues/source?bid=2452379&fileBranchId=2878156#l28)